### ISSUE TITLE
Prediction of store sales

### ISSUE THREAD

**Issue by Haylemicheal** [Haylemicheal/Pharmaceutical-Sales-Prediction#4]
- Preprocessing
- Building models with sklearn pipelines
- Choose a loss function
- Post Prediction analysis
- Serialize models
- Building model with deep learning
- Using MLFlow to serve the prediction
